web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1375,11 +1376,282 @@ article footer {
   font-style: italic;
 }
 
+/* BP - Job Post Page */
+
 .form-item label {
-  font-size: 14px;
+  font-size: 20px;
   font-weight: bold;
+  font-family: Montserrat;
+  line-height: 2em;
 }
 
+.form-item input[type=text], .form-item input[type=email], .form-item input[type=password], .form-item input[type=search], .form-item input[type=tel], .form-item textarea, .form-item select {
+    display: block;
+    width: 100%;
+    max-width: 1200px;
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 8px 0;
+}
+
+input#edit-label-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+textarea#edit-description-summary-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+form#job-role-post-form {
+    padding-left: 20px;
```

Comment:
class rather than id if possible. Is this also something we want to be on all forms?